src/pybind/mgr/dashboard/frontend/src/app/ceph/rgw/rgw-multisite-zonegroup-form/rgw-multisite-zonegroup-form.component.html

```diff
@@ -89,6 +89,94 @@
+            <div class="card"
+                 [formGroup]="item">
+              <div class="card-header">
+                {{ (index + 1) | ordinal }}
+                <span class="float-end clickable"
+                      name="remove_placement_target"
+                      (click)="removePlacementTarget(index)"
+                      ngbTooltip="Remove">&times;</span>
+              </div>
+
+              <div class="card-body">
+                <!-- Placement Id -->
+                <div class="form-group row">
+                  <label i18n
+                         class="cd-col-form-label required"
+                         for="placement_id">Placement Id</label>
+                  <div class="cd-col-form-input">
+                    <input type="text"
+                           class="form-control"
+                           name="placement_id"
+                           id="placement_id"
+                           formControlName="placement_id"
+                           placeholder="eg. default-placement">
+                    <span class="invalid-feedback">
+                      <span *ngIf="showError(index, 'placement_id', formDir, 'required')"
+                            i18n>This field is required.</span>
+                    </span>
+                  </div>
+                </div>
+
+                <!-- Tags-->
+                <div class="form-group row">
+                  <label i18n
+                         class="cd-col-form-label"
+                         for="tags">Tags</label>
+                  <div class="cd-col-form-input">
+                    <input type="text"
+                           class="form-control"
+                           name="tags"
+                           id="tags"
+                           formControlName="tags"
+                           placeholder="comma separated tags, eg. default-placement, ssd">
+                  </div>
+                </div>
+
+                <!-- Storage Class -->
+                <div class="form-group row">
+                  <label i18n
+                         class="cd-col-form-label"
+                         for="storage_class">Storage Class</label>
+                  <div class="cd-col-form-input">
+                    <input type="text"
+                           class="form-control"
+                           name="storage_class"
+                           id="storage_class"
+                           formControlName="storage_class"
+                           placeholder="eg. Standard-tier">
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </ng-container>
+        <button type="button"
+                id="add-plc"
+                class="btn btn-light float-end my-3"
+                (click)="addPlacementTarget()">
+          <i [ngClass]="[icons.add]"></i>
+          <ng-container i18n>Add placement target</ng-container>
+        </button>
+      </div>
```

Comment:
This is super weird because I cannot create with placement groups but then I have to somehow, go to edit, and find out you can add placements now.

This should be added to the create form also. If your worries are that the create command doesn't have this feature to add placement targets, then the backend can do it in two steps, create and then add those placements.

Reply:
There are default placement targets that are created when we create a zonegroup, so its not mandatory or necessary for the user to create more placement targets at the time of zonegroup creation. So, to keep the creation workflow as simple as possible and keeping the user from encountering so many input fields at the time of creation, we have moved it to the edit workflow

Reply:
You can check the greed workflow here - https://tracker.ceph.com/attachments/download/4124/rgw%20multi-site%204%20of%206%20(pages%2025-34)%20-%20edit%20multisite%20(inc%20placement%20target%20and%20storage%20class).pdf